operator/src/main/java/org/bf2/operator/managers/StrimziManager.java

```diff
@@ -118,7 +141,7 @@ private boolean isStrimziDeployment(Deployment deployment) {
     }
 
     private void updateStatus() {
-        List<StrimziVersionStatus> versions = new ArrayList<>(this.strimziVersions.values());
+        List<StrimziVersionStatus> versions = this.strimziVersions.values().stream().map(ComponentVersions::getStrimziVersion).collect(Collectors.toCollection(ArrayList::new));
```

Comment:
Just a minor thing, you don't need to guarantee mutability here, so it could just be Collectors.toList.